### PR TITLE
bundle csv service with webpack

### PR DESCRIPTION
**Pull request by BCaspari**

### CODE REVIEW

File: csv/main.ts [hunk 1]
```diff
@@ -0,0 +1,2 @@
+export * from "./csvreader-config";
```
Comment (JPressmar) [inline csv/main.ts:1]: Does it work like this? Would be perfect if it does.

File: csv/package.json [hunk 2]
```diff
@@ -6,14 +6,15 @@
...
+    "build": "tsc && npm run build:server && npm run build:tests && npm run webpack && rimraf tmp",
+    "build:server": "babel . --out-dir tmp --config-file ./../server.babel.json --extensions .ts,.tsx,.js --ignore \"./**/*.d.ts\",\"./**/*.roxtest.ts\",\"./**/*.roxselenium.ts\",\"./node_modules/**/*\",\"./dist/**/*\",\"./disttests/**/*\",\"./webpack.config.js\"",
     "build:tests": "rimraf ./disttests && cross-env BABEL_ENV=debug babel . --out-dir disttests --config-file ./../server.babel.json --extensions .ts,.tsx,.js --ignore \"./**/*.d.ts\",\"./node_modules/**/*\",\"./dist/**/*\",\"./disttests/**/*\"",
     "test": "cross-env HOTRELOAD=no DEBUG=bpmn-engine:123* mocha ./disttests/**/*.roxtest.js --exit --recursive --timeout 40000 --require regenerator-runtime --require source-map-support/register --require ./../commontestfiles/registerbabel.js",
     "copyproduction": "cross-var copyfiles package.json production && cd ./production && npm install --production && cd .. && cross-var copyfiles -u 1 \"./production/node_modules/**/*\" $npm_package_servicename && rimraf production/",
```
Comment (JPressmar) [inline csv/package.json:13]: copyproduction is obolete now and should be removed.

**Review by JPressmar — CHANGES_REQUESTED** [review]

File: csv/main.ts (hunk 1 above)
Comment (BCaspari) [reply csv/main.ts:1]: yes, it works

File: csv/package.json [hunk 3]
```diff
@@ -6,14 +6,14 @@
...
+    "build:server": "babel . --out-dir tmp --config-file ./../server.babel.json --extensions .ts,.tsx,.js --ignore \"./**/*.d.ts\",\"./**/*.roxtest.ts\",\"./**/*.roxselenium.ts\",\"./node_modules/**/*\",\"./dist/**/*\",\"./disttests/**/*\",\"./webpack.config.js\"",
     "build:tests": "rimraf ./disttests && cross-env BABEL_ENV=debug babel . --out-dir disttests --config-file ./../server.babel.json --extensions .ts,.tsx,.js --ignore \"./**/*.d.ts\",\"./node_modules/**/*\",\"./dist/**/*\",\"./disttests/**/*\"",
     "test": "cross-env HOTRELOAD=no DEBUG=bpmn-engine:123* mocha ./disttests/**/*.roxtest.js --exit --recursive --timeout 40000 --require regenerator-runtime --require source-map-support/register --require ./../commontestfiles/registerbabel.js",
-    "copyproduction": "cross-var copyfiles package.json production && cd ./production && npm install --production && cd .. && cross-var copyfiles -u 1 \"./production/node_modules/**/*\" $npm_package_servicename && rimraf production/",
     "copyfiles": "cross-var copyfiles service.json dist/* $npm_package_servicename && npm run copyproduction",
```
Comment (JPressmar) [inline csv/package.json:13]: This script calls `npm run copyproduction` remove it there too.

**Review by JPressmar — APPROVED** [review]

**Pull request merged**